scenario.py

```diff
@@ -53,8 +54,11 @@ def __init__(self, params, experiment_path):
             self.corrupted_nodes = ["not_corrupted"] * self.nodes_count
 
         # Define if test data should be distributed between nodes...
+
+        # ... or if each node should refer to a centralised test set...
+        # ... when training on a single node
         # ... or if each node should refer to a centralised test set
-        self.testset_option = params["testset_option"]
+        self.testset_option = params["testset_option"] # Toggle between 'Centralised' and 'Distributed'
```

Comment:
Does it make this clearer like this : ?

> When training on a single node, the test set can be either the local node test set or the global test set

Then I would rename `testset_option `to `single_test_set_evaluation `= `local `or `global`

Reply:
Ok, changed and clarified in commit ec27f20.